[2017/04/12 15:21:20] :: [pool-1-thread-19] :: Base path for the test = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\<br><br>[2017/04/12 15:21:20] :: [pool-1-thread-19] :: Current Thread Name for the test is "pool-1-thread-19"<br><br>[2017/04/12 15:21:33] :: [pool-1-thread-19] :: The user has launched 'http://test.salesforce.com<br><br>[2017/04/12 15:21:53] :: [pool-1-thread-19] :: arindom.das@move.com.qa has successfully loggedin<br><br>[2017/04/12 15:21:56] :: [pool-1-thread-19] :: The user has creating NewBrokerAccountorder<br><br>[2017/04/12 15:21:58] :: [pool-1-thread-19] :: Successfully navigated to Home page<br><br>[2017/04/12 15:21:59] :: [pool-1-thread-19] :: File path is = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\ .<br><br>[2017/04/12 15:22:21] :: [pool-1-thread-19] :: Successfully New order account ID Fetched= readSFInputData .<br><br>[2017/04/12 15:22:27] :: [pool-1-thread-19] :: Successfully Account ID is visible= 001j000000YrYNb .<br><br>[2017/04/12 15:22:52] :: [pool-1-thread-19] :: Successfully clicked on GlobalSearchBtn .<br><br>[2017/04/12 15:23:32] :: [pool-1-thread-19] :: Successfully clicked on PeopleName .<br><br>[2017/04/12 15:23:55] :: [pool-1-thread-19] :: Successfully clicked on FollowDown .<br><br>[2017/04/12 15:24:10] :: [pool-1-thread-19] :: Successfully clicked on UserDetail .<br><br>[2017/04/12 15:24:30] :: [pool-1-thread-19] :: Successfully clicked on LoginUsers .<br><br>[2017/04/12 15:24:45] :: [pool-1-thread-19] :: Successfully Account Name is visible 001j000000YrYNb .<br><br>[2017/04/12 15:25:04] :: [pool-1-thread-19] :: Broker Account doesn't have any parent account<br><br>[2017/04/12 15:26:39] :: [pool-1-thread-19] :: Successfully selected office and created primary account for broker account<br><br>[2017/04/12 15:27:26] :: [pool-1-thread-19] :: parent account id  is not visible<br><br>[2017/04/12 15:27:30] :: [pool-1-thread-19] :: Exception occurred. C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\OutPutData.xlsx (The process cannot access the file because it is being used by another process)<br><br>[2017/04/12 15:27:34] :: [pool-1-thread-19] :: Closing the Browser Instance<br><br>[2017/04/12 15:27:34] :: [pool-1-thread-19] ::  File Name: null Method name: open(-2)<br> File Name: null Method name: <init>(-1)<br> File Name: null Method name: <init>(-1)<br> File Name: WriteExcelFile.java Method name: writeExcel(67)<br> File Name: NonInventory.java Method name: creatingParentAccountWithOffice(4534)<br> File Name: C:/New Selenium/Salesforce_QA_Automation/ADT_Resources/SalesForce/FeatureFiles/CREATE_NEW_BROKER_ACCOUNTS/Copy of Creating_New_Broker_Account.feature Method name: Then creating parent account with office(11)<br><br>Complete Error Message: C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\OutPutData.xlsx (The process cannot access the file because it is being used by another process)<br><br><br>